stdlib/2/tokenize.pyi

```diff
@@ -102,15 +103,15 @@ chain = ...  # type: type
 double3prog = ...  # type: type
 endprogs = ...  # type: Dict[str, Any]
 pseudoprog = ...  # type: type
-re = ...  # type: module
+re = ...  # type: ModuleType
```

Comment:
This probably shouldn't be here at all. (I'm guessing tokenize just imports re.)